lib/view_model/send/send_view_model.dart

```diff
@@ -61,8 +61,9 @@ abstract class SendViewModelBase with Store {
   ObservableList<Output> outputs;
 
   @action
-  void addOutput() {
-    outputs.add(Output(_wallet, _settingsStore, _fiatConversationStore, () => selectedCryptoCurrency));
+  void addOutput({String? currency}) {
+    outputs.add(Output(_wallet, _settingsStore, _fiatConversationStore,
+        currency != null ? () => getCryptoCurrency(currency) : () => selectedCryptoCurrency));
```

Comment:
here it can be always `() => selectedCryptoCurrency` and you just have to call `setSelectedCryptoCurrency` before adding the output